features/features.go

```diff
@@ -68,6 +68,9 @@ const (
 	// V1DisableNewValidations disables validations for new domain names in the V1
 	// API.
 	V1DisableNewValidations
+	// PrecertificateOCSP ensures that we write an OCSP response immediately upon
+	// generating a precertificate.
```

Comment:
I think it might be worth trying to mention the change in serial number generation as well so that if there were a problem with the new `AddSerial` RPC it would be easier to conceptually link it to this feature flag.

Reply:
Done :+1: